components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -19,9 +19,13 @@
   "web_accessible_resources": [
   ],
   "background": {
-    "scripts": ["out/brave_extension_background.bundle.js"]
+    "scripts": ["out/brave_extension_background.bundle.js"],
+    "persistent": true
```

Comment:
Why do we need this? If it uses an API that needs the background script to be alive won't that just happen automatically?

Reply:
It's needed by the `webRequest` API. I'm not sure if it will stay alive automatically, it's called out in the docs to make it persisent here:  https://developers.chrome.com/extensions/background_pages ("The webRequest API is incompatible with non-persistent background pages.")